Stripe/STPAPIClient+ApplePay.m

```diff
@@ -126,7 +126,7 @@ + (NSDictionary *)parametersForPayment:(PKPayment *)payment {
     payload[@"pk_token"] = paymentString;
     payload[@"card"] = [self addressParamsFromPKContact:payment.billingContact];
 
-    NSCAssert(!(paymentString.length == 0 && [[Stripe defaultPublishableKey] hasPrefix:@"pk_live"]), @"The pk_token is empty. Using Apple Pay with an iOS Simulator while not in Stripe Test Mode will always fail.");
+    NSCAssert(!(paymentString.length == 0 && [[STPAPIClient sharedClient].publishableKey hasPrefix:@"pk_live"]), @"The pk_token is empty. Using Apple Pay with an iOS Simulator while not in Stripe Test Mode will always fail.");
```

Comment:
I think this should still reference [Stripe defaultPublishableKey]`

Reply:
As it stands, `STPAPIClient sharedClient` is still more likely to be accurate of the two, since if they differ, it means the user set `STPAPIClient.sharedClient.publishableKey`.

e.g. You could never call `Stripe.setDefaultPublishableKey` and only set `STPAPIClient.sharedClient.publishableKey`.